test/fixtures/service-account-with-quota.json

```diff
@@ -0,0 +1,13 @@
+{
+    "type": "service_account",
+    "project_id": "my-account",
+    "private_key_id": "abc123",
+    "private_key": "-----BEGIN RSA PRIVATE KEY-----\nMIICXAIBAAKBgQDI+tbB7kfmWFsD6Y+GUzRazzE62JbajO806zYVJrIx9TsmPx/E\nzfHVJy2hlONaSOfKJD7udT1w4bVo+OpWkMvXYAiqZFg+4EAg+FvuBzToyD2VyiqZ\n6hOXT3xVzPFLJDwpATm7SvGmzIbhAHLOH6QBePEw1ADhyXW2sHZbzzwEmwIDAQAB\nAoGBAJcmTy06j0hlWs3uccqL+OvytwuSqKFlLOGXo+z0VT/NNtbk0neoix/Lfz3u\nQ6469lfIOqwL8FFc7por2dGQxDu42yO2FVSaKkgNKZz2BIv64sZkKiSvMB/eScBe\nRBsUClBd828jShDlAaOucyoDqajop9EfiuysBPjSLo7Li4MBAkEA6h/tFUeyM0hv\n5OuoZU+gUTCVBGePaBFjarS1ufq8XiIIG9UPDIDGaYnBjWYJmodojwiigJ7glidc\nxV0yfu3kywJBANvCHT7QAXrnTw7xPMin0VbazUujf7fgTSpUM5/CMhI9AlAJZhx8\np8xPJG9mOd44lFS765IwHupVLqA5sPgUNXECQH3cAdihvUNiWnymztT/tEBRLJq8\njVQ1nMs9MEA1cVPtWYyUwc1H4OHVY05/HwSKbIQP8UPRpQwzRuT2j/G6M3cCQCbj\nEaPE0Q47kjzVqWjkcWHKNBvXYcla8qyz27LAfXXGv/sDvsL5uAOqWYsw7rqJDo5z\n9nqW81GKI0cNDmjHwrECQGnF6BaF1SvKw/kbKeF34OVw5FQOmmGZP5y3b49wC8Rp\nhGrm3z7yM8jiQIywqpv5KDBKKMiKMR5pIcbxscalY4k=\n-----END RSA PRIVATE KEY-----\n",
```

Comment:
I generated this private key in place for tests.

Reply:
I ended up using the key generation library already in tests to reduce confusion.